src/apb/engine.py

```diff
@@ -321,6 +337,14 @@ def update_dockerfile(project):
     print('Finished writing dockerfile.')
 
 
+def load_source_dependencies(roles_path):
+    print('Trying to guess list of dependencies for APB')
+    output = subprocess.check_output("/bin/grep -R \ image: "+roles_path+"|awk '{print $3}'", stderr=subprocess.STDOUT, shell=True)
```

Comment:
By choosing which images are referenced in the k8s ansible modules or the openshift modules. If the APB specifies container images then we want to be able to intuitively pick them out.